### PR TITLE
Update Dial-Once Android SDK information

### PR DESCRIPTION
**Pull request by CAMOBAP795**

### CODE REVIEW

File: android/specs/index.md [hunk 1]
```diff
@@ -32,24 +32,23 @@ Dial Once uses these intents to launch itself and to perform tasks such as displ
...
-Currently (as of v2.3.1), the Dial Once SDK has a total weight of *149KB*.
+Currently (as of v2.6.3), the Dial Once SDK has a total weight of *155KB*.
 
 > This is indicative and may change in the near future as we are constantly updating and adding new features.
 
 What are the underlying dependencies?
 ------------------------------------------
 
-Current version (v2.3.1) have the following dependency tree:
+Current version (v2.6.3) have the following dependency tree:
```
Comment (Spring3) [inline android/specs/index.md]: ``has`` instead of ``have``

Comment (CAMOBAP795) [inline android/specs/index.md]: Fixed

**Comment by CAMOBAP795** [comment]
@jkernech I think make sense to merge it once we will upload 2.6.3 to 'stable' channel on bintray.com, right?

**Comment by jkernech** [comment]
@CAMOBAP795 Exactly 👍

**Comment by jkernech** [comment]
@CAMOBAP795 can you update the doc with the latest version of the SDK (v2.6.4) please

**Comment by CAMOBAP795** [comment]
@jkernech sorry for late I will cancel this PR and create a new one with correct branch version (to keep clean git history)

**Comment by CAMOBAP795** [comment]
@jkernech  new PR created https://github.com/dialonce/dialonce.github.io/pull/6

**Pull request closed without merging**